src/core/server/legacy/legacy_service.ts

```diff
@@ -253,7 +253,9 @@ export class LegacyService implements CoreService {
         )
       : EMPTY;
 
-    require('../../../cli/cluster/cluster_manager').create(
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    const ClusterManager = require('../../../cli/cluster/cluster_manager');
+    return new ClusterManager(
```

Comment:
Using the public constructor instead of the static `create`, as we already have the legacy-plugin-mutated config and want to use this one.

Reply:
Does that mean we can remove the static factory function `create` or is it still being used somewhere?

Reply:
We might be able to do that. will grep and see